x-pack/plugins/security/server/lib/privileges/privileges.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function getVersionPrivilege(kibanaVersion) {
+  // TODO: Remove the .toLowerCase() once the capitalization with app privileges is fixed
+  return `version:${kibanaVersion.toLowerCase()}`;
+}
+
+export function getLoginPrivilege() {
+  return `action:login`;
+}
+
+export function buildPrivilegeMap(savedObjectTypes, application, kibanaVersion) {
+  const readSavedObjectsPrivileges = buildSavedObjectsReadPrivileges(savedObjectTypes);
+
+  const privilegeActions = {};
```

Comment:
This are the "app privileges" right?

Reply:
No good reason at this point, I will make it so.